pkg/config.go

```diff
@@ -39,6 +39,32 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy by this value.
+// The acceptable value MUST BE either`TLS_1_1`/`TLS_1_2`/`TLS_1_3`
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
+	tlsVersions := []string{"TLS_1_1", "TLS_1_2", "TLS_1_3"}
+	for _, ver := range tlsVersions {
+		if os.Getenv("MIN_TLS_VERSION") != ver || os.Getenv("MIN_TLS_VERSION") == "" {
+			return "TLS_1_2"
+		}
+	}
+	return os.Getenv("MIN_TLS_VERSION")
```

Comment:
Repetitive things aren't look good. Also there's LookupEnv func that could be used for this kind of job.

Reply:
Since we use multiple input parameters, It's not recommended to use the `os.LookupEnv`

> os.LookupEnv() returns the value of the environment variable in its first parameter if set, otherwise the second parameter is false. Allows distinguishing unset from empty value.